src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -493,6 +493,7 @@ describe('ReactDOMComponent', function() {
           return nodeValue;
         },
         set: nodeValueSetter.mockImplementation(function(newValue) {
+          node.setAttribute('value', newValue);
```

Comment:
Said offline but the DOM doesn't do this so while it tricks the test it's not real life.